src/Definition.php

```diff
@@ -127,57 +108,36 @@ public function hasMethod($method)
      * Get a given method definition
      *
      * @param  string $method
-     * @return null|\Laminas\Server\Method\Definition
+     * @return false|Method\Definition
```

Comment:
`false` is not a type, use `bool` instead.

Reply:
Done.